src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialOfficeAppointmentProcessor.java

```diff
@@ -39,34 +39,29 @@ public void process(Exchange exchange) {
         List<JudicialOfficeAppointment> filteredJudicialAppointments = validate(judicialOfficeAppointmentJsrValidatorInitializer,
                 judicialOfficeAppointments);
 
-        log.info("Judicial Appointment Records count after Validation:: " + filteredJudicialAppointments.size());
+        List<JudicialUserProfile> invalidJudicialUserProfileRecords = judicialUserProfileJsrValidatorInitializer.getInvalidJsrRecords();
+
+        filterInvalidUserProfileRecords(filteredJudicialAppointments, invalidJudicialUserProfileRecords);
 
```

Comment:
this should log teh records which are skipped from appointment table in exception table as well